src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -83,7 +83,26 @@ private Program(Options options)
             }
 
             var newBranchName = await MakePrBranch(_options.SourceUser, _options.RepoName, remoteIntoBranch, newBranchPrefix);
-            await SubmitPullRequest(newBranchName);
+            var pullRequest = await SubmitPullRequest(newBranchName);
+
+            // pullRequest could be null if we are running in debug mode.
+            // Only write the comment if the pull request can be automatically merged.
+            if (pullRequest != null && (pullRequest.Mergeable ?? false))
```

Comment:
Is `Mergable` a `bool?`?  If so, I'd prefer `if (pullRequest?.Mergable.HasValue && pullRequest.Mergable.Value)` rather than have the `??`, but I don't care enough to block this PR.